src/visions/backends/shared/parallelization_engines.py

```diff
@@ -45,8 +41,6 @@ def setup(cls, *args, **kwargs) -> None:
         if cls._is_setup:
             return
 
-        import swifter
```

Comment:
Any reason to disable swifter in it's entirety?

Reply:
Mistake on my end. It's a pity this PR is now so old but IIRC I was trying to see if I could roll any numba improvements in as an engine. Should be fixed now.